### PR TITLE
Converts ms to s

### PR DESCRIPTION
**Pull request by NejcZdovc**

### CODE REVIEW

File: src/ledger_impl.cc [hunk 1]
```diff
@@ -265,7 +265,7 @@ void LedgerImpl::GetPublisherInfoList(uint32_t start, uint32_t limit,
     prefix, callback);
 }
 
-void LedgerImpl::SetPublisherMinVisitTime(uint64_t duration) { // In milliseconds
+void LedgerImpl::SetPublisherMinVisitTime(unsigned int duration) { // In seconds
```
Comment (bridiver) [inline src/ledger_impl.cc]: this should still be uint64_t to match Time::InSeconds

File: include/bat/ledger/ledger.h [hunk 2]
```diff
@@ -95,7 +95,7 @@ class LEDGER_EXPORT Ledger {
                                     const ledger::PublisherInfoFilter& filter,
                                     GetPublisherInfoListCallback callback) = 0;
 
-  virtual void SetPublisherMinVisitTime(uint64_t duration_in_milliseconds) = 0;
+  virtual void SetPublisherMinVisitTime(uint64_tb duration_in_seconds) = 0;
```
Comment (bridiver) [inline include/bat/ledger/ledger.h]: tb?

Comment (NejcZdovc) [reply include/bat/ledger/ledger.h]: yeah, fixes just now

Comment (bridiver) [reply include/bat/ledger/ledger.h]: I don't think seconds or milliseconds really matters to ledger. As long as the client uses one or the other consistently everything should be correct either way.

Comment (bridiver) [reply include/bat/ledger/ledger.h]: seconds is fine for now, but we could change it to `ticks` and let the client decide

Comment (NejcZdovc) [reply include/bat/ledger/ledger.h]: yeah point of this PR is to sync it all across, so that we only use one type

**Pull request merged**